packages/create-plugin/src/constants.ts

```diff
@@ -99,6 +100,10 @@ export const TEXT = {
   updateNpmDependenciesSuccess: 'Successfully updated the NPM dependencies.',
   updateNpmDependenciesAborted: 'No NPM dependencies have been updated.',
 
+  updateProvisioning: '**Would you like to add provisioning files?**',
```

Comment:
Since this is asking about add, the locale variable could be better named `addProvisioning`